resource_customizations/tower.ansible.com/AnsibleJob/health.lua

```diff
@@ -0,0 +1,25 @@
+hs = {}
+if obj.status ~= nil then
+  if obj.status.ansibleJobResult ~= nil then
+    jobstatus = obj.status.ansibleJobResult.status
+    if jobstatus == "successful" then
+      hs.status = "Healthy"
+      hs.message = obj.status.ansibleJobResult.url
+      return hs
+    end
+    if jobstatus == "failed" or jobstatus == "error" or jobstatus == "canceled" then
+      hs.status = "Degraded"
+      hs.message = obj.status.ansibleJobResult.url
```

Comment:
```suggestion
      hs.message =. "job " .. jobstatus .. " - " .. obj.status.ansibleJobResult.url
```

Reply:
Excellent suggestion! 
I am going with the following for now. Please let me know what you think. Cheers.
```
hs.message = jobstatus .. " job - " .. obj.status.ansibleJobResult.url
```